task/10-katas-1-tasks.js

```diff
@@ -113,7 +184,45 @@ function getZigZagMatrix(n) {
  *
  */
 function canDominoesMakeRow(dominoes) {
-    throw new Error('Not implemented');
+    let temp = dominoes.shift(),
+         i = 0;
+    while (dominoes.length != 0)
+    {
+        if (dominoes[i][0] == temp[0]) 
+        {
+            temp = dominoes[i].reverse() + temp;
+            dominoes.splice(i, 1);
+            i = 0;
+            continue;
+        }
+        if (dominoes[i][1] == temp[0]) 
+        {
+            temp = dominoes[i] + temp;
+            dominoes.splice(i, 1);
+            i = 0;
+            continue;
+        }
+        if (dominoes[i][0] == temp[1]) 
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"